src/test/java/com/cvsgo/controller/TagControllerTest.java

```diff
@@ -0,0 +1,94 @@
+package com.cvsgo.controller;
+
+import com.cvsgo.dto.tag.TagResponseDto;
+import com.cvsgo.entity.Tag;
+import com.cvsgo.service.TagService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.restdocs.RestDocumentationContextProvider;
+import org.springframework.restdocs.RestDocumentationExtension;
+import org.springframework.restdocs.payload.JsonFieldType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import org.springframework.web.filter.CharacterEncodingFilter;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static com.cvsgo.ApiDocumentUtils.getDocumentRequest;
+import static com.cvsgo.ApiDocumentUtils.getDocumentResponse;
+import static org.mockito.BDDMockito.given;
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.document;
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.documentationConfiguration;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.get;
+import static org.springframework.restdocs.payload.PayloadDocumentation.fieldWithPath;
+import static org.springframework.restdocs.payload.PayloadDocumentation.responseFields;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.setup.SharedHttpSessionConfigurer.sharedHttpSession;
+
+@ExtendWith(RestDocumentationExtension.class)
+@WebMvcTest(TagController.class)
+public class TagControllerTest {
```

Comment:
JUnit5 부터는 접근제한자를 public으로 선언하지 않아도 돼서 삭제해도 될 것 같아요!~
```suggestion
class TagControllerTest {
```